test/unit/base.test.js

```diff
@@ -879,6 +880,23 @@ describe('CONVERT FUNCTION TESTS ', function() {
         });
       });
     });
+
+    it('Should correctly define URL for root server with base URL variables', function (done) {
+      var openapi = fs.readFileSync(rootUrlServerWithVariables, 'utf8');
+      Converter.convert({ type: 'string', data: openapi }, {}, (err, conversionResult) => {
+        let requestUrl,
+          collectionVars;
+        expect(err).to.be.null;
+        expect(conversionResult.result).to.be.true;
+
+        requestUrl = conversionResult.output[0].data.item[0].request.url;
+        collectionVars = conversionResult.output[0].data.variable;
+        expect(requestUrl.host).to.eql(['{{baseUrl}}']);
```

Comment:
what would requestUrl.host have been before this fix? Same for the other 2 expects

Reply:
Before fix `requestUrl.host` was `{{BASE_URI}}` and generated request contained entire `{BASE_URI}/api/primary-domain/works` as URL which is not similarly structured to `{{baseUrl}}/path_in_schema` URL structure we follow in other cases.

For other two expects behaviour was already same as collection variables was generated regardless. We are just checking that after resolving of variables, correct URL is to be present.